docs/guides/how-to/general.md

```diff
@@ -0,0 +1,48 @@
+Here are several pointers to other places in the docs, for general or frequent questions.
+
+
+### Set a Service Name
+To see your service names in the logfire UI, you just need to set the `service_name` parameter in
+[`logfire.configure()`][logfire.configure(service_name)].
+
+### Understanding the SQL Schema
+With Logfire, you can use the UI to run arbitrary SQL queries against your trace and metric data to
+analyze and investigate your system. For details of the data model, see
+our [schema docs](../web-ui/explore.md#records-schema)
+
+### Disable Logfire During Tests
```

Comment:
The linked testing page starts by saying that `send_to_logfire=False` by default under pytest.

I know this was asked about in slack at some point, but this behaviour was implemented since then.